week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,143 @@
-/** Write your CSS in here **/
+body {
+  background-color: #0352fc;
+}
+#autogenerator{
+  background-color: #c8ccc9;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: x-large;
+  margin-left: 25%;
+  margin-right: 25%;
+  font-family:'Rubik Mono One', sans-serif;
+}
+
+#container {
+  background-color: #c8ccc9;
+  height: 20em;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  margin-top: 12em;
+  margin-left: 10em;
```

Comment:
Avoid setting a specific `margin-left` if what you want to achieve is a centered element. 
Set `margin-left` and `margin-right` to `auto` and the element should be centered. Alternatively, wrap it all inside a flex container and set it to `justify-content: center`

Reply:
(for this to work, the flex container would have to be the width of the screen, and the grey box would need to be **less** wide than its flex container.)